src/Plugin/GraphQL/DataProducer/Entity/EntityLoadMultiple.php

```diff
@@ -31,7 +31,6 @@
  *     ),
  *     "language" = @ContextDefinition("string",
  *       label = @Translation("Entity languages"),
```

Comment:
We should also fix the label here to "Entity language"

Reply:
Label updated